unstructured/ingest/main.py

```diff
@@ -384,6 +415,21 @@ def main(
                 re_download=re_download,
             ),
         )
+    elif biomed_path or biomed_api_id or biomed_api_from or biomed_api_until or biomed_api_format:
```

Comment:
up in the block around line 309 under

        if not download_dir:
            cache_path = Path.home() / ".cache" / "unstructured" / "ingest"
           ...

A case needs to be added for biomed so that downloads may properly be stored under ~/.cache (which saves files from being re-download with the user having to think about that too much).

While your at it, you could change this very confusing [ValueError](https://github.com/Unstructured-IO/unstructured/blob/c51adb2/unstructured/ingest/main.py#L299) to something like:

`This connector does not support saving downloads to ~/.cache/  , --download-dir must be provided`